pipelines/LSSTComCam/ApPipe.yaml

```diff
@@ -9,6 +9,10 @@ imports:
     include:
       - prompt
 tasks:
+  isr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      doBrighterFatter: False
```

Comment:
Could you add a comment saying that this will be merged into `ap_pipe` on... I guess it's still DM-47750?

Reply:
Thanks. Adding a comment is a good idea. 

Yes I think just using DM-47750 is good enough; no need for more paperwork :p